---
title: "Class_exercise"
author: Corrin Bowman
date: 3/20/2023
output: github_document
---

# Load the libraries you need
# Load functions you need "my_class_functions"
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)


library(GenomicRanges)
library(tidyverse)
library(IRanges)
source("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/util/BCHM5631_my_class_functions.R")


```


# load in your peak files for each replicate of each protein
# Here I am starting to analyze my data for my proteins of interest:
# proteinX, Y, Z .....
# First I will read in each replicate file
```{r load in peak files}

# filepath to import peaks
basepath <- "/scratch/Shares/rinnclass/CLASS_2023"
peak_path <- "2023_2_10_group3/group/results/bwa/mergedLibrary/macs/broadPeak/"
broadpeakfilepath <- file.path(basepath, peak_path)

# import peaks
peak_list <- import_peaks(consensus_file_path = broadpeakfilepath)

# let's get a list of how many peaks are in each file before we create consensus peaks.
peak_num <- sapply(peak_list, length) %>% as.data.frame(row.names = T)
# label column
names(peak_num) <- c("num_peaks")


# printing out a table of the number of peaks in each file:
peak_num <- peak_num %>%
  rownames_to_column(var = "dbp") %>%
  separate(col = dbp,  into = c('dbp', 'replicate'), sep = "_")

```


# Now I am going to create consensus peaks for each protein
```{r consensus peaks}

dbps <- unique(sapply(names(peak_list), function(x) {
   unlist(strsplit(x, "_"))[1]
}))

consensus_list <- lapply(dbps, consensus_from_reduced, peak_list)
names(consensus_list) <- dbps

num_consensus_peaks <- sapply(consensus_list, length) %>% 
  as.data.frame() %>%
  rownames_to_column( var = "dbp") %>%
  dplyr::rename(number_consensus_peaks = ".")

peak_num <- left_join(peak_num, num_consensus_peaks)

# export consensus peaks to results folder

write_csv(peak_num, "results/num_peaks_df.csv")

```

# Now I am going to make my consensus peaks compatable with UCSC genome browser
```{r}

basepath <- "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman"
consensus_path <- "CLASS_2023/consensus peaks"
exportpath <- file.path(basepath, consensus_path)

for(i in 1:length(consensus_list)) {
rtracklayer::export(consensus_list[[i]], paste0(exportpath, names(consensus_list)[i], "_consensus_peaks.bed") )}

consensus_file_list <- list.files("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/00_consensus_peaks/Consensus Peaks", full.names = T, pattern = ".bed")

peaks <- lapply(consensus_file_list, read.table, col.names = c("chr", "start", "end", "name", "score", "strand"))

names(peaks) <- dbps

canonical_chr <- c(paste0("chr", 1:22), "chrM", "chrX", "chrY")
peaks <- lapply(peaks, function(x) x %>% filter(chr %in% canonical_chr))

new_filenames <- paste0("consensus peaks/", names(peaks), "_consensus.bed")

for(i in 1:length(peaks)) {
  write.table(peaks[[i]], new_filenames[[i]],
              sep = "\t", col.names = FALSE, row.names = FALSE,
              quote = FALSE, append = TRUE)
}

headers <- paste0("track type=bedGraph name=", names(peaks))
headers


# print out consensus peak files in a results/UCSC directory

new_filenames <- paste0("ucsc_consensus_peaks/", names(peaks), ".bed")
new_filenames

for(i in 1:length(peaks)) {
  # Write the header line
  writeLines(headers[[i]], new_filenames[[i]])
  # Append the broadPeak table data
  
  write.table(peaks[[i]], new_filenames[[i]],
              sep = "\t", col.names = FALSE, row.names = FALSE,
              quote = FALSE, append = TRUE)
}

```


# I am curious if my proteins are transcription factors so I will use the annotations
# in a cell paper I found and see

```{r}
url <- "https://www.cell.com/cms/10.1016/j.cell.2018.01.029/attachment/ede37821-fd6f-41b7-9a0e-9d5410855ae6/mmc2.xlsx"

destination_for_url <- "results/TF_annotations.xlsx" 

# to download we can use download.file
download.file(url, destination_for_url)


human_tfs <- readxl::read_excel("results/TF_annotations.xlsx",
                                sheet = 2, skip = 1)


# let's rename the 4th column to indicate if it is a TF.
names(human_tfs)[4] <- "is_tf"
num_peaks_df <- data.frame("dbp" = names(consensus_peaks),
                           "num_peaks" = sapply(consensus_peaks, length))

length(which(tolower(num_peaks_df$dbp) %in% tolower(human_tfs$Name)))

human_tfs <- human_tfs[tolower(human_tfs$Name) %in% tolower(num_peaks_df$dbp), 1:4]


# adding new column names
names(human_tfs) <- c("ensembl_id",
                      "dbp",
                      "dbd",
                      "tf")
num_peaks_df <- merge(num_peaks_df, human_tfs, all.x = T) 

# Let's check how many NAs -- we should have some missing values  
dim(num_peaks_df[is.na(num_peaks_df$tf),])

```




# Now I want to compare a protein with a previous analysis 
```{r}

# goto UCSC genome browser and load in a peak file for a given protein
# load in the data for the same protein from the previous analysis
# compare how your consensus peaks are similar or different to previous analyses


#I uploaded EP300 and compared this to the EP300 from spring of 2021. I believe what I was looking at was had mostly similar peaks with minor differences. 

```


# Now I am going to determine how my peaks for each protein overlap annotations of the genome
# First I will find the overlaps between my consensus peaks with promoters of lncRNA and mRNA promoters

```{r}

library(tidyverse)
library(GenomicRanges)
source("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/util/BCHM5631_my_class_functions.R")

basepath <- "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman"
peak_path <- "CLASS_2023/CLASSES/05_R_analyses/00_consensus_peaks/Consensus Peaks"
consensusPeakPath <- file.path(basepath, peak_path)


 consensus_peaks_files<- list.files(consensusPeakPath, 
                                             pattern = "*.bed",
                                             full.names = TRUE)

consensus_peaks <- lapply(consensus_peaks_files, rtracklayer::import)

names(consensus_peaks) <- gsub("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/00_consensus_peaks/Consensus Peaks/|_consensus.bed","", consensus_peaks_files)

gencode_gr <- rtracklayer::import("/scratch/Shares/rinnclass/CLASS_2023/data/data/genomes/gencode.v32.annotation.gtf")

gencode_genes <- gencode_gr[gencode_gr$type == "gene"] 
table(gencode_gr$type)

rtracklayer::export(gencode_genes, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/01_peak_features/results2/gene_annotations/gencode_genes.gtf")


mrna_genes <- gencode_genes[gencode_genes$gene_type %in% "protein_coding"] 

rtracklayer::export(mrna_genes, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/01_peak_features/results2/gene_annotations/mrna_genes.gtf")
table(gencode_genes$gene_type)

lncrna_genes <- gencode_genes[gencode_genes$gene_type %in% "lncRNA"] 

rtracklayer::export(lncrna_genes, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/01_peak_features/results2/gene_annotations/lncrna_genes.gtf")


mrna_lncrna_genes <- gencode_genes[gencode_genes$gene_type %in% c("protein_coding","lncRNA")]
rtracklayer::export(mrna_lncrna_genes, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/01_peak_features/results2/gene_annotations/mrna_lncrna_genes.gtf")



lncrna_mrna_genes <- rtracklayer::import("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/01_peak_features/results2/gene_annotations/mrna_lncrna_genes.gtf")

# Nice that loaded so much faster -- lets see as DF
lncrna_mrna_genes_df <- lncrna_mrna_genes %>% as.data.frame()


lncrna_mrna_promoters <- promoters(lncrna_mrna_genes, upstream = 1000, downstream = 1000)
# check right size
width(lncrna_mrna_promoters) # 2000 
rtracklayer::export(lncrna_mrna_promoters, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/01_peak_features/results2/gene_annotations/lncrna_mrna_promoters.gtf")


# last handy annotation will be lncRNA and mRNA gene IDs to subset
lncrna_gene_ids <- mrna_lncrna_genes$gene_id[mrna_lncrna_genes$gene_type == "lncRNA"]
table(mrna_lncrna_genes$gene_type)
# same for mRNAs
mrna_gene_ids <-mrna_lncrna_genes$gene_id[mrna_lncrna_genes$gene_type == "protein_coding"]

num_peaks_df <- data.frame("dbp" = names(consensus_peaks),
                           "num_peaks" = sapply(consensus_peaks, length))


# Now let's get the total amount of the genome covered by all the peaks for a given DBP.

num_peaks_df$total_peak_length <- sapply(consensus_peaks, function(x) sum(width(x)))

source ("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/util/my_class_functions.R")
promoter_peak_counts <- count_peaks_per_feature(lncrna_mrna_promoters, consensus_peaks, type = "counts")

num_peaks_df$peaks_overlapping_promoters <- rowSums(promoter_peak_counts)



```

## results: 
#1) What can you determine from these overlaps?

Here we looked at the overlaps between my consensus peaks with promoters of lncRNA and mRNA promoters. 

CEBPZ- most overlaps were with mRNA promoters and very few were with lncRNA 

CHD2- 2nd highest amount of overlaps for both lncRNA and mRNA promoters- with more being mRNA
 
CTCF- 3rd highest amount of overlaps for both lncRNA and mRNA promoters- with more being mRNA

ELF1- greatest amount of overlaps with both lncRNA and mRNA promoters

EP300- least amount of overlaps with both lncRNA and mRNA promoters

 


# Now I want to compare the overlaps with lncRNA and mRNA promoters seperately 
```{r}

num_peaks_df$peaks_overlapping_lncrna_promoters <- rowSums(promoter_peak_counts[,lncrna_gene_ids]) 

# mrna promoter overlaps
num_peaks_df$peaks_overlapping_mrna_promoters <- rowSums(promoter_peak_counts[,mrna_gene_ids])

write_csv(num_peaks_df, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/01_peak_features/results2/num_peaks_df.csv")

# answer is in the file ^ 


```
## results:
# 1) What is the difference in overlaps between mRNA and lncRNA promoters
For my proteins the majority of the peak overlaps are more mRNA compared to lncRNA promoters- besides EP300 it has more overlaps with lncRNA.


# Now I am going to test if there is more binding over gene bodies than promoters
# I will seperate lncRNA and mRNA gene bodies to find the overlaps 

```{r}

source ("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/util/my_class_functions.R")
promoter_peak_counts <- count_peaks_per_feature(lncrna_mrna_promoters, consensus_peaks, type = "counts")

num_peaks_df$peaks_overlapping_promoters <- rowSums(promoter_peak_counts)

genebody_peak_counts <- count_peaks_per_feature(mrna_lncrna_genes, 
                                                consensus_peaks, 
                                                type = "counts")

num_peaks_df$peaks_overlapping_genebody <- 
  rowSums(genebody_peak_counts)

# lncRNA gene bodies 
num_peaks_df$peaks_overlapping_lncrna_genebody <- rowSums(genebody_peak_counts[,lncrna_gene_ids])

# mRNA gene bodies
num_peaks_df$peaks_overlapping_mrna_genebody <- 
  rowSums(genebody_peak_counts[,mrna_gene_ids])
write_csv(num_peaks_df, "num_peaks_df.csv")


```
## results: 
# 1) Do my proteins have more overlaps with promoters or genebodies?


CEBPZ- promoters

CHD2- genebody
 
CTCF- genebody by a lot 

ELF1- genebody

EP300- genebody 


# It is nice and all to find overlaps, but I am interested in how many proteins
# bind a specific promoter. I will use my handy "occurence" parameter in 
# " count peaks per feature" 

```{r}

promoter_peak_occurence <- count_peaks_per_feature(lncrna_mrna_promoters, consensus_peaks, 
                                               type = "occurrence")
stopifnot(all(colnames(promoter_peak_occurence) == lncrna_mrna_promoters$gene_id))

write.table(promoter_peak_occurence, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/01_peak_features/results2/lncrna_mrna_promoter_peak_occurence_matrix.tsv")

```


```{R turnign peak_occurence matrix into data.frame}


stopifnot(all(colnames(promoter_peak_occurence) == lncrna_mrna_promoters$gene_id))

peak_occurence_df <- data.frame("gene_id" = colnames(promoter_peak_occurence),
                                "gene_name" = lncrna_mrna_promoters$gene_name,
                                "gene_type" = lncrna_mrna_promoters$gene_type,
                                "chr" = lncrna_mrna_promoters@seqnames,   
                                "1kb_up_tss_start" = lncrna_mrna_promoters@ranges@start,
                                "strand" = lncrna_mrna_promoters@strand,
                                "number_of_dbp" = colSums(promoter_peak_occurence))

# "counts" just gave us value for the dbp of interest

write_csv(peak_occurence_df, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/01_peak_features/results2/peak_occurence_dataframe.csv")


```
## results: I find the max number of proteins on a promoter to be X

10

# Now I want to start plotting my results
# First I will see if there is a realtionship between peak number and total DNA covered
```{r}

library(ggplot2)
library(tidyverse)

source("../../../util/plotting_functions.R")
source("../../../util/_setup.R")

num_peaks_df <- read_csv('/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/01_peak_features/results/num_peaks_df.csv')


ggplot(num_peaks_df, aes(x = num_peaks, 
                         y = total_peak_length)) +
  geom_point() 

```

# Now I want to color my plot by wether the protein is a TF or not.
```{r}


ggplot(num_peaks_df, aes(x = num_peaks, 
                 y = total_peak_length,
                   color= tf )
                ) +
  geom_point() 



ggplot
```

# I want to make a histogram of the number of peaks for each of my proteins

```{r}

ggplot(num_peaks_df, aes(x = num_peaks)) +
  geom_histogram()

hist
```


# Now I want to facet this by the type of DNA binding domain my protein has.
```{r}

ggplot(num_peaks_df, aes(x = num_peaks, 
                 y = total_peak_length
                )) +
  facet_wrap(dbd ~ .) +
  geom_point() 


```


# Cool now I am ready to send my result to my collaborator as a
# Knitted document


