---
title: "Big_Knit"
author: "Corrin Bowman"
date: "4/22/2023"
output: github_document
editor_options: 
chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
options(stringsAsFactors = F)
library(tidyverse)
library(GenomicRanges)
library(ggplot2)
source("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/util/my_class_functions.R")
source("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/util/_setup.R")
library(pheatmap)

```

# Loading in all .broadPeak files for all dbps and their replicates

```{r load .broadPeak files}

basepath <- "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman"
peak_path <- "CLASS_2023/CLASSES/05_R_analyses/analysis/00_consensus_peaks"
broadpeakfilepath <- "/scratch/Shares/rinnclass/CLASS_2023/data/data/peaks"

# using import peaks to import .broadPeak files (~10min)
peak_list <- import_peaks(consensus_file_path = broadpeakfilepath)

save(peak_list, file= "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/peak_list.Rdata")

```

# creating consensus peaks
Using consensus_from_reduced function that requires peak list
and dbp object. This will be done in 2 steps
(1) created dbp object of unique names for dbp
(2) run consensus_from_reduced


```{r create consensus peaks}

  # Creating unique DBP object for create_consensus_peaks_from_reduced
dbp <- unique(sapply(names(peak_list), function(x) {
   unlist(strsplit(x, "_"))[1]
}))

# now run our function consensus_from_reduced
consensus_list <- lapply(dbp, consensus_from_reduced, peak_list)

# adding names to the GRange list
names(consensus_list) <- dbp

save(consensus_list, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/consensus_list.RData")

load("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/consensus_list.RData", verbose = T)
```

# exploring the number of peaks in the consensus_list
```{r exploring number of peaks in consenus peaks}

# creating list of num_peaks per dbp
num_peaks <- sapply(consensus_list, length)

# plotting
hist(num_peaks, breaks = 1000)
hist(num_peaks, breaks = 1000, xlim = c(0,3000))

```
Result: seems like 1000 peaks should be the min moving forward, Filter out any chip data less 1,000 peaks == filtered consensus peaks

# filtering consensus_list to dbps with > 1000 peaks
```{r filtered_consenus_list}

# filtering to 1000 peaks
filtered_consensus_list <- consensus_list[sapply(consensus_list, length) > 1000]

# saving 
save(filtered_consensus_list, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/filtered_consensus_list.RData")

# keeping track of DBPs lost
lost_dbps <- names(consensus_list[sapply(consensus_list, length) < 1000]) %>% as.data.frame()

# saving 
write.table(lost_dbps, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/lost_dbps.csv")

```

# exporting filtered_consensus_peaks
```{r exporting filtered consensus peaks}

for(i in 1:length(filtered_consensus_list)) {
  rtracklayer::export(filtered_consensus_list[[i]], 
                      paste0("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/filtered_consensus_peaks/", 
                             names(filtered_consensus_list)[i], 
                             "_filtered_consensus_peaks.bed"))
}


#TODO rename so can load into UCSC.

#consensus_file_list <- list.files("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/Consensus Peaks", full.names = T, pattern = ".bed")

#peaks <- lapply(consensus_file_list, read.table, col.names = c("chr", "start", "end", "name", "score", "strand"))

#names(peaks) <-dbp

#canonical_chr <- c(paste0("chr", 1:22), "chrM", "chrX", "chrY")
#peaks <- lapply(peaks, function(x) x %>% filter(chr %in% canonical_chr))

# now that these are all nice and clean let's export: pasting a string together to name files
#new_filenames <- paste0("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/Consensus Peaks/", names(peaks), "_consensus.bed")

#for(i in 1:length(peaks)) {
 # write.table(peaks[[i]], new_filenames[[i]],
 #             sep = "\t", col.names = FALSE, row.names = FALSE,
 #             quote = FALSE, append = TRUE)
#}

#headers <- paste0("track type=bedGraph name=", names(peaks))

#new_filenames <- paste0("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/ucsc_consensus_peaks/", names(peaks), ".bed")


#for(i in 1:length(peaks)) {
  #writeLines(headers[[i]], new_filenames[[i]])

  #write.table(peaks[[i]], new_filenames[[i]],
             #sep = "\t", col.names = FALSE, row.names = FALSE,
            #quote = FALSE, append = TRUE)
#}

```

# loading in genome features
```{r creating genome feature objects}

gencode_gr <- rtracklayer::import("/scratch/Shares/rinnclass/CLASS_2023/data/data/genomes/gencode.v32.annotation.gtf")

# gencode genes
gencode_genes <- gencode_gr[gencode_gr$type == "gene"] 

# mrna_genes
mrna_genes <- gencode_genes[gencode_genes$gene_type %in% "protein_coding"]

# lncrna_genes
lncrna_genes <- gencode_genes[gencode_genes$gene_type %in% "lncRNA"] 

# mrna_lncrna_genes
mrna_lncrna_genes <- gencode_genes[gencode_genes$gene_type %in% c("protein_coding","lncRNA")]

# lncrna_mrna_promoters
lncrna_mrna_promoters <- promoters(mrna_lncrna_genes, upstream = 1000, downstream = 1000)

# lncrna_gene_ids
lncrna_gene_ids <- mrna_lncrna_genes$gene_id[mrna_lncrna_genes$gene_type == "lncRNA"]

# mrna_gene_ids
mrna_gene_ids <-mrna_lncrna_genes$gene_id[mrna_lncrna_genes$gene_type == "protein_coding"]

```


# making data frame of filtered_consensus_peak info
```{r creating num_peaks_df to track peak properties}

num_peaks_df <- data.frame("dbp" = names(filtered_consensus_list),
                           "num_peaks" = sapply(filtered_consensus_list, length))

# total genome covered by peaks
num_peaks_df$total_peak_length <- sapply(filtered_consensus_list, function(x) sum(width(x)))

# creating number of promoter overlaps entry
promoter_peak_counts <- count_peaks_per_feature(lncrna_mrna_promoters, filtered_consensus_list, type = "counts")

# creating promoter peak_occurence for clustering - Metaplots later.
promoter_peak_matrix <- count_peaks_per_feature(lncrna_mrna_promoters, filtered_consensus_list, type = "occurrence")

# saving
write.table(promoter_peak_matrix, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/promoter_peak_occurrence_matrix.tsv")

# read back in
promoter_peak_occurrence_matrix <- read.table("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/promoter_peak_occurrence_matrix.tsv")

save(promoter_peak_occurrence_matrix, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/promoter_peak_occurrence_matrix.RData")

# summing rows to get total number of promoter overlaps
num_peaks_df$peaks_overlapping_promoters <- rowSums(promoter_peak_counts)

# lncrna promoter overlaps 
num_peaks_df$peaks_overlapping_lncrna_promoters <- rowSums(promoter_peak_counts[,lncrna_gene_ids])

# mrna promoter overlaps
num_peaks_df$peaks_overlapping_mrna_promoters <- rowSums(promoter_peak_counts[,mrna_gene_ids])

# Finding overlaps with gene_bodies (will take a few minutes again)
# Note this takes several minutes
genebody_peak_counts <- count_peaks_per_feature(mrna_lncrna_genes, 
                                                filtered_consensus_list, 
                                                type = "counts")

# All gene bodies overlaps
num_peaks_df$peaks_overlapping_genebody <- rowSums(genebody_peak_counts)

# lncRNA gene bodies 
num_peaks_df$peaks_overlapping_lncrna_genebody <- rowSums(genebody_peak_counts[,lncrna_gene_ids])

# mRNA gene bodies
num_peaks_df$peaks_overlapping_mrna_genebody <- rowSums(genebody_peak_counts[,mrna_gene_ids])


```

# adding TF type annotations for DBPs
```{r adding TF features to num_peaks_df}

# download TF annotations to results
#url <- "https://www.cell.com/cms/10.1016/j.cell.2018.01.029/attachment/ede37821-fd6f-41b7-9a0e-9d5410855ae6/mmc2.xlsx"
#destination_for_url <- "results/TF/TF_annotations.xlsx"

# to download we can use download.file
#download.file(url, destination_for_url)

# reading in TF annotations 
human_tfs <- readxl::read_excel("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/TF/TF_annotations.xlsx",
                                sheet = 2, skip = 1)

# let's rename the 4th column to indicate if it is a TF.
names(human_tfs)[4] <- "is_tf"

# now let's intersect gene names that are in our ChIP data and has TF identity.
length(which(tolower(num_peaks_df$dbp) %in% tolower(human_tfs$dbp)))
# 407 of the 430 have matching gene_names - not bad

human_tfs <- human_tfs[tolower(human_tfs$Name) %in% tolower(num_peaks_df$dbp), 1:4]
# adding new column names
names(human_tfs) <- c("ensembl_id",
                      "dbp",
                      "dbd",
                      "tf")

# merging into num_peaks_df
num_peaks_df <- merge(num_peaks_df, human_tfs, all.x = T)


```
# How many of these proteins are TFs? What is the most represented type of DBD?

table(num_peaks_df$tf)  

When I run the code indexing into the num_peaks file and then the tf column- there are 63 no and 344 yes

table(human_tfs$dbd)

C2H2 ZF is the most represented type of DBD and is seen 199 times- significantly more than the rest






# creating promoter peak occurence matrix
This will make a matrix where promoters are cols (30K)
Each will have 1 if overlapped by a given dbp : 0 if no overlaps

```{r promoter peak occurence matrix}

# running count_peaks_per_feature
promoter_peak_occurence <- count_peaks_per_feature(lncrna_mrna_promoters, filtered_consensus_list, 
                                               type = "occurrence")

# Let's double check that all lncrna & mrna genes are accounted for:
stopifnot(all(colnames(promoter_peak_occurence) == lncrna_mrna_promoters$gene_id))

# saving
write.table(promoter_peak_occurence, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/lncrna_mrna_promoter_peak_occurence_matrix.tsv")

# Now let's use the 'data.frame()' fucntion. Set up a bunch of colnames and populate them.
peak_occurence_df <- data.frame("gene_id" = colnames(promoter_peak_occurence),
                                "gene_name" = lncrna_mrna_promoters$gene_name,
                                "gene_type" = lncrna_mrna_promoters$gene_type,
                                "chr" = lncrna_mrna_promoters@seqnames,   
                                "1kb_up_tss_start" = lncrna_mrna_promoters@ranges@start,
                                "strand" = lncrna_mrna_promoters@strand,
                                "number_of_dbp" = colSums(promoter_peak_occurence))

# saving
write_csv(peak_occurence_df, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/peak_occurence_dataframe.csv")


```

# now make a promoter data_frame that tells which dbps are bound
```{r dbp centric promoter occurence}

# dbps on promoters object
DBPs_on_promoter <- lncrna_mrna_promoters %>%
                    as.data.frame() %>%
  dplyr::select(gene_id, gene_name)

# creating promoter dbps by pivot longer of promoter_peak_occurence_matrix
promoter_dbps <- promoter_peak_occurence %>%
  as.data.frame() %>%
  rownames_to_column("dbp") %>%
pivot_longer(2:ncol(.), names_to = "gene_id", values_to = "occurrence") %>%
  filter(occurrence == 1) %>%
  dplyr::select(-occurrence) %>%
  left_join(DBPs_on_promoter)

save(promoter_dbps, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/promoter_dbps.RData")

# checking Firre promoter
firre_promoter <- promoter_dbps %>%
  filter(gene_name == "FIRRE")

# XIST promoter (should be off since male cells)
XIST_promoter <- promoter_dbps %>%
  filter(gene_name == "XIST")

# GAPDH
GAPDH_promoter <- promoter_dbps %>%
  filter(gene_name == "GAPDH")

# saving
promoter_dbps_df <- promoter_dbps %>% as.data.frame()
write.csv(promoter_dbps, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/promoter_dbps.csv")

save(promoter_dbps, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/promoter_dbps.RData")

```

# saving key env objects
```{r saving key objects in environment to load next session}

save(filtered_consensus_list, gencode_genes, lncrna_gene_ids, mrna_gene_ids, num_peaks_df, peak_occurence_df, promoter_peak_occurrence_matrix, lncrna_mrna_promoters, mrna_lncrna_genes, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/peak_features.RData")
# awesome - now we never have to load in consensus peaks again :)

#saving other things 
save(DBPs_on_promoter, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/DBPs_on_promoter.RData")
save(lost_dbps, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/lost_dbps.RData")



```





# loading in peak_features env objects from 01_create_consensus_peaks
```{r laod env objects}

load("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/peak_features.RData", verbose = T)

```


# Peaks per dbp
```{r plotting peak features}

# First let's look at a histogram of peak#/DBP
 ggplot(num_peaks_df, aes(x = num_peaks)) + 
  geom_histogram(bins = 70)
# saving
ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/02_plotting_consensus_peaks/figures/num_peaks_hist.pdf")

```


# plotting num_peaks versus genome coverage.
```{r peaks vs coverage}

ggplot(num_peaks_df, aes(x = num_peaks, y = total_peak_length)) +
  geom_point() + 
  geom_smooth(method = "gam", se = TRUE, color = "black", lty = 2)+
  ylab("BP covered") +
  xlab("Number of peaks") +
  ggtitle("Peak count vs. total bases covered")

# saving 
ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/02_plotting_consensus_peaks/figures/peak_num_vs_coverage.pdf")


```
# How does peak number and genome coverage compare

looking at the plot of peak counts vs. total genome- there seems to be a correlation when the peak number is high the genome coverage is also high for example- ZNF280B, when the number of peaks is low the genome coverage is also lower. It looks pretty linear in the graph with some outliers. The genome coverage is a way bigger number, magnitudes bigger, than the peak numbers. This makes sense because the whole genome coverage isn't going to be covered in peaks.


# plotting num peaks on promoters

```{r number of DBPS on promoters}

ggplot(num_peaks_df,
       aes(x = num_peaks, y = peaks_overlapping_promoters)) +
  xlab("Peaks per DBP") +
  ylab("Number of peaks overlapping promoters") +
  ggtitle("Relationship Between Number of DBP Peaks and Promoter Overlaps")+
  geom_point() +
  geom_abline(slope = 1, linetype="dashed") +
  geom_smooth(method = "lm", se=FALSE, formula = 'y ~ x',
              color = "#a8404c") +
 # stat_regline_equation(label.x = 35000, label.y = 18000) +
  ylim(0,60100) +
  xlim(0,60100)

ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/02_plotting_consensus_peaks/figures/peak_num_vs_promoter_coverage.pdf")

# peaks you stop increaseing binding to promoters.
# maybe it leaks into the gene body let's check
```
Result: saturation of binding events -- as you get more peaks 
you stop increasing binding to promoters -- probably saturated.

# peak Coverage on gene bodies
```{r peak coverage on gene bodies}

ggplot(num_peaks_df,
       aes(x = num_peaks, y = peaks_overlapping_genebody)) +
  xlab("Peaks per DBP") +
  ylab("Number of peaks overlapping genes") +
  ggtitle("Relationship Between Number of DBP Peaks and Gene Body Overlaps")+
  geom_point() +
  geom_abline(slope = 1, linetype="dashed") +
  geom_smooth(method = "lm", se=F, formula = 'y ~ x',
              color = "#a8404c") +
  #stat_regline_equation(label.x = 35000, label.y = 18000) +
  ylim(0,60100) +
  xlim(0,60100)

# saving
ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/02_plotting_consensus_peaks/figures/4_peak_num_vs_gene_body_coverage.pdf")
```
Result: Gene bodies explain almost all the places of binding in the genome

# What is the distribution of promoter overlaps versus gene-bodies (hint hist)

hist(num_peaks_df$peaks_overlapping_promoters, breaks = 1000)
hist(num_peaks_df$peaks_overlapping_genebody, breaks = 1000)

When I look at the distribution of promoter overlaps it seems to be more evenly distributed across the x-axis. The majority of frequencies being 1, a lot are unique.

When I look at the distribution of gene body overlaps it is skewed to the left but much higher values than the promoter overlaps. The majority of frequencies are 1 meaning there is a lot of uniqueness.


# Density plot of binding events
Let's make a density plot of num DBPs bound per promoter

```{r density plot of DBP localization events}

ggplot(peak_occurence_df, aes(x = number_of_dbp)) +
geom_density(alpha = 0.2, color = "#424242", fill = "#424242") +
  theme_paperwhite() +
  xlab(expression("Number of DBPs")) +
  ylab(expression("Density")) +
  ggtitle("Promoter binding events",
          subtitle = "mRNA and lncRNA genes") 

# saving
ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/02_plotting_consensus_peaks/figures/num_binding_events_per_promoter.pdf")

```
Result: very interesting that the promoter binding is bimodal !
Result: most promoters have upto 100 dbps then a lag and 2nd dist at ~200dbs
# RESULT: There are two types of promoter binding - (1) normal (2) super-binders

# Make a list of genes that are "super binders" 
I am filtering the peak_occurence_df file in the number of dbp column above 200 to be considered a super binder which means over 200 dbps bind. 

super= filter(peak_occurence_df, number_of_dbp > 200) 
super$gene_name

There are so many names that are super binders which can be seen from the above code. 

# Is there a type of gene ontology associated with them versus the others?

I am going to index into the super binders list and see what kind of gene type they are and how many- 

table(super$gene_type)

What I found was there are two gene types associated with the super binders- lnRNA (2512) and protein coding (9184)


# promoters with out binding events
Lets find how many promoters don't have any DBPs bound
```{r prmoters with out binding events}

unbound_promoters <- peak_occurence_df %>% 
  filter(peak_occurence_df$number_of_dbp < 1)

# how many are there?
nrow(unbound_promoters)
# so there are only a few 6,720 promoters that don't have binding evetns (~10%)

#  let's put it in a folder called results. We will always use this folder structure
write_csv(unbound_promoters, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/02_plotting_consensus_peaks/results/unbound_promoters.csv")

```

# lncRNA versus mRNA promoter binding
Let's compare the binding patterns of lncRNA vs mRNA promoters.
```{r lncrna vs mrna promoter binding}

ggplot(num_peaks_df, aes(x = num_peaks)) +
  geom_point(aes(y = peaks_overlapping_lncrna_promoters), color = "red") +
  geom_point(aes(y = peaks_overlapping_mrna_promoters), color = "black") +
  #stat_regline_equation(aes(y = peaks_overlapping_lncrna_promoters), color = "red") +
  #stat_regline_equation(aes(y = peaks_overlapping_mrna_promoters), color = "black", label.y = 20000) +
  geom_smooth(aes(y = peaks_overlapping_lncrna_promoters), method = "lm", se = FALSE, formula = "y ~ x") +
  geom_smooth(aes(y = peaks_overlapping_mrna_promoters), method = "lm", se = FALSE, formula = "y ~ x")

# saving
ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/02_plotting_consensus_peaks/figures/lncRNA-mRNA-promoter_binding.pdf")



# This is just a fancier version of the same thing -- note mutate to clean up names
num_peaks_dfl <- num_peaks_df %>%
  dplyr::select(-peaks_overlapping_promoters) %>%
  pivot_longer(cols = peaks_overlapping_lncrna_promoters:peaks_overlapping_mrna_promoters,
               names_to = "gene_type",
               values_to = "peaks_overlapping_promoters") %>%
  mutate(gene_type = gsub("peaks_overlapping_", "", gene_type))

# plotting
ggplot(num_peaks_dfl, aes(x = num_peaks, y = peaks_overlapping_promoters, 
                         col = gene_type)) +
         geom_point() +
         geom_abline(slope = 1, linetype="dashed") +
  geom_smooth(method = "lm", se = FALSE, formula = "y ~ x") +
  #stat_regline_equation() +
  scale_color_manual(values = c("#a8404c", "#424242"))+
  xlab("Peaks per DBP") +
  ylab("Peaks Overlapping Promoters") +
  ggtitle("Number of DBP Peaks and Promoter Overlaps")

# saving
ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/02_plotting_consensus_peaks/figures/peaks_overlaps_relationship_by_gene_type.pdf", height = 5, width = 8)


```
# Is there a difference in mRNA and lncRNA promoter overlaps?

When looking at the plot of number of DBPs and promoter overlaps for mRNA and lncRNA- mRNA definetely has more peaks overlapping promoters and has a greater slope than lncRNA. lncRNA may have less but seems more linear relationship than mRNA. There is 9,184 mRNA superbinders. Superbinders are more closely related to mRNA vs. lncRNA.


# Do lncRNAs also have super-binding promoters?

Yes they still have super binders there are 2512 superbinders but not as many as mRNA has.





#03


# creating distance matrix & dendrogram
```{r distance matrix and dendrogram}

# creating distance matrix
peak_occurence_dist <- dist(promoter_peak_occurrence_matrix, method = "binary")

# clustering distance values
bin_hier <- hclust(peak_occurence_dist, method = "complete")

# Dendrogram of binding profiles by promoter (not binding profile - below)
ggdendro::ggdendrogram(bin_hier, rotate = FALSE,  size = 3,
                       theme_dendro = TRUE) +
   coord_flip() +
   scale_y_continuous() +
   scale_x_continuous(position = "top") +
   scale_x_continuous(breaks = seq_along(bin_hier$labels[bin_hier$order]),
             labels = bin_hier$labels[bin_hier$order], position = "top",
             expand = c(0,0)) +
   theme(axis.text.x = element_text(angle = 90, hjust  = 1)) +
   theme(axis.text.y = element_text(angle = 0,hjust = 1)) +
   scale_y_reverse(expand = c(0.01, 0)) +
   theme(
     plot.background = element_blank(),
     panel.grid.major = element_blank(),
   panel.grid.minor = element_blank(),
     panel.border = element_blank()
   )

ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/promoter_overlap_dendrogram.pdf")

```

# Using profile_tss for all 430 DBPs
# ! this takes ~45 min !
```{r metaplot DF of binding profiles over promoter}

# establishing DF
metaplot_df <- data.frame(x = integer(), dens = numeric(), dbp = character())

# for loop to populate DF 
for(i in 1:length(filtered_consensus_list)) {
  print(names(filtered_consensus_list)[[i]])
  tmp_df <- profile_tss(filtered_consensus_list[[i]], lncrna_mrna_promoters)
  tmp_df$dbp <- names(filtered_consensus_list)[[i]]
  metaplot_df <- bind_rows(metaplot_df, tmp_df)
  
}

# saving
write_rds(metaplot_df, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/results/metaplot_df_final.rds")


save(metaplot_df, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/results/metaplot_df_final.RData")


```
  

# creating distance matrix of binding profile correlations
```{r scaling and plotting dendrogram of binding similarity by promoter}

metaplot_filtered_matrix <- metaplot_df %>% 
  pivot_wider(names_from = x, values_from = dens) %>%
  column_to_rownames("dbp") %>%
  as.matrix()
mm_scaled <- t(scale(t(metaplot_filtered_matrix)))
metaplot_hclust <- hclust(dist(mm_scaled), method = "complete")

# plotting relationship between binding profiles
plot(metaplot_hclust)
pdf("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/tss_profile_dendrogram.pdf", height = 10, width = 27)
par(cex=0.3)
plot(metaplot_hclust)
dev.off()
```

# What genes make sense to cluster together create an image (hint Pol II makes RNA)

I filtered through to get all the different gene types and found-  processed_pseudogene, lncRNA , unprocessed_pseudogene, miRNA, protein_coding.

It makes sense to cluster genes together that have similar functions/roles. For example all the protein coding genes together, and transcription factors together. 


# Find a cluster of genes your interested in and can figure out what those genes do -- are there unknown genes i there too? If so maybe one could hypothesize they have a similar function to the known genes (Hint ZNFs)

I looked at a smaller cluster with a few genes that are protein coding such as- smc3, suz12. Also in the cluster toward the right side is a few genes that are related to transcription factors such as- nfe2i2, maff and cebpb. There is a zinc finger gene(ZNF460) right next to the transcription factors and in this cluster so I guessed it was a function related to transcription factors but this gene is actually a protein coding one- so related to the other half of the cluster. 

Another example= ZNF883 is realted to cell carcinoma and is right next to gene EGR1 which is also associated with cancer- Leukemia.

I searched through a lot of the zinc finger genes and found one example that didn't seem to have much information on what it really does- ZNF614 is described as a protein coding gene realted to pathways for gene expression. The spot next door is gabpb1  which is protein coding related to gene expression. This one is involved in encoding the GA-binding protein transcription factor, beta subunit so maybe ZNF614 could be for something similiar with a beta subunit. 
 

# heat map of binding profile over +/- 1kb TSS window
```{r heatmap of profile over TSS (+/- 1Kb)}

# setting up PDF function for saving

# complex heatmap of mm_scaled

pdf("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/tss_profile_heatmap.pdf", height = 35, width = 10)
pheatmap(mm_scaled, cluster_columns = FALSE, border = TRUE, 
        show_column_names = FALSE,
        use_raster = TRUE,
        column_split = split,
        column_gap = unit(0, "mm"))
dev.off()
```

# looking more into the patterns of ChIP binding profiles relative to TSS
# breaking this down into several clusters and plotting each
```{r how many groups are there?}


# Let's see how many DBPs have different patterns.
# manually checking at 6 we get a new group of 10 - after that not much changes
clusters <- cutree(metaplot_hclust, k=6)
table(clusters)

# plot cluster 1
dev.new()
pdf("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/cluster_1_heatmap.pdf", height = 35, width = 10)

pheatmap(mm_scaled[clusters == 1,], cluster_columns = FALSE, border = TRUE, 
        show_column_names = FALSE,
        use_raster = TRUE,
        column_split = split,
        column_gap = unit(0, "mm"),)
graphics.off()

#row_names_gp = gpar(fontsize = 7)
# TSS enriched a bit broader than cluster 2


# cluster 2

dev.new()
pdf("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/cluster_2_heatmap.pdf", height = 35, width = 10)

pheatmap(mm_scaled[clusters == 2,], cluster_columns = FALSE, border = TRUE, 
        show_column_names = FALSE,
        use_raster = TRUE,
        column_split = split,
        column_gap = unit(0, "mm"),)
graphics.off()

#row_names_gp = gpar(fontsize = 7)
# TSS enrichred a bit more narrow

# cluster 3
dev.new()
pdf("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/cluster_3_heatmap.pdf", height = 35, width = 10)

pheatmap(mm_scaled[clusters == 3,], cluster_columns = FALSE, border = TRUE, 
        show_column_names = FALSE,
        use_raster = TRUE,
        column_split = split,
        column_gap = unit(0, "mm"))
graphics.off()

#row_names_gp = gpar(fontsize = 7)
# very narrow binding over tss !

# cluster 4

dev.new()
pdf("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/cluster_4_heatmap.pdf", height = 35, width = 10)
pheatmap(mm_scaled[clusters == 4,], cluster_columns = FALSE, border = TRUE, 
        show_column_names = FALSE,
        use_raster = TRUE,
        column_split = split,
        column_gap = unit(0, "mm"))
graphics.off()

#row_names_gp = gpar(fontsize = 7)
# looks like TSS depletion and histone mods

# cluster 5 only 1 DBP
names(clusters[5])


# cluster 6
dev.new()
pdf("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/cluster_6_heatmap.pdf", height = 35, width = 10)
pheatmap(mm_scaled[clusters == 6,], cluster_columns = FALSE, border = TRUE, 
        show_column_names = FALSE,
        use_raster = TRUE,
        column_split = split,
        column_gap = unit(0, "mm"))
graphics.off()

# TSS depletion and histone mods again.


```

# establishing lncRNA and mRNA promoters (+/- 1kb)
```{r create lncRNA and mRNA promoters }

# creating promoters just in case:
lncrna_promoters <- lncrna_mrna_promoters[lncrna_mrna_promoters$gene_type == "lncRNA"]
mrna_promoters <- lncrna_mrna_promoters[lncrna_mrna_promoters$gene_type == "protein_coding"]


```

# metaplots for each DBP by lncRNA and mRNA promoters
```{r}

#setting up lncrna DF.
lncrna_metaplot_df <- data.frame(x = integer(), dens = numeric(), dbp = character())

# for loop to populate DF with overlap density in lncrna promoters
for(i in 1:length(filtered_consensus_list)) {
  print(names(filtered_consensus_list)[[i]])
  tmp_df <- profile_tss(filtered_consensus_list[[i]], lncrna_mrna_promoters = lncrna_promoters)
  tmp_df$dbp <- names(filtered_consensus_list)[[i]]
  lncrna_metaplot_df <- bind_rows(lncrna_metaplot_df, tmp_df)
  
}

# saving
write_rds(lncrna_metaplot_df, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/results/lncRNA_metaplot_df_final.rds")

save(lncrna_metaplot_df, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/results/lncRNA_metaplot_df_final.RData")


# now for mRNAs 
mrna_metaplot_df <- data.frame(x = integer(), dens = numeric(), dbp = character())

# for loop to populate mRNA_metaplot
for(i in 1:length(filtered_consensus_list)) {
  print(names(filtered_consensus_list)[[i]])
  tmp_df <- profile_tss(filtered_consensus_list[[i]], lncrna_mrna_promoters = mrna_promoters)
  tmp_df$dbp <- names(filtered_consensus_list)[[i]]
  mrna_metaplot_df <- bind_rows(mrna_metaplot_df, tmp_df)
  
}

# saving mRNA metaplots
write_rds(mrna_metaplot_df, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/results/mrna_metaplot_df_final.rds")

save(mrna_metaplot_df, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/results/mrna_metaplot_df_final.RData")


# now adding the information of gene type
mrna_metaplot_df$gene_type <- "mRNA"
lncrna_metaplot_df$gene_type <- "lncRNA"
combined_metaplot_profile <- bind_rows(mrna_metaplot_df, lncrna_metaplot_df)

# saving
write_rds(mrna_metaplot_df, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/results/metaplot_df_final.rds")


#TODO why not printing
# TODO asign cluster value and plot the average of the clusters
# plotting - NOTE facet wrap by dbp !

# pdf(file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/mega_plot_test.pdf")
ggplot(combined_metaplot_profile, 
       aes(x = x, y = dens, color = gene_type )) +
  geom_vline(xintercept = 0, lty = 2) + 
  geom_line(size = 1.5) + 
  facet_wrap(dbp ~ ., scales = "free_y") +
  ggtitle("Promoter Metaplot") + 
  scale_x_continuous(breaks = c(-1000, 0, 1000),
                     labels = c("-1kb", "TSS", "+1kb"),
                     name = "") + 
  ylab("Peak frequency") +
 scale_color_manual(values = c("#424242","#a8404c"))

# saving
ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/mega_meta_plot_lncRNA-mRNA.pdf", width = 49, height = 12)
# ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/03_clustering/figures/mega_meta_plot_lncRNA-mRNA.pdf", limitsize = F)


```
# if we cluster by lncRNA and mRNA separately what are some similarities and differences?

Some similarities are they are both clustering so they have similiar shapes/figure. 
They both are pretty far apart/spread out- the clustering height is high
One difference is the mRNAs seem to be more closely related to eachother because they are closer vs. the lncRNAs seem slightly less related.

# Let's look at the metaplot for all DBPs on lncRNA and mRNA promoters seperately (hint facet wrap).
Mega-Mega plot 

# Which genes seem to have a difference in where they bind on promoters between lncRNA and mRNA promoters

Looking at the big facet we have every gene in a plot with lncRNA as black and mRNA as red. Majority of all the graphs look similiar, a negative parabola with both lncRNA and mRNA pretty much overlapping eachother in the same spot/shape. There are a few exceptions that I noticed- 

1. H3K36me3- the shape was not a parabola but a bunch of squiggles, and lncRNA and mRNA don't overlap eachother really
2. H3K4me1, H3K79me2, H4K20me1- the parabola is flipped facing upwards rather down and lncRNA doesn't go down as far- the overlap is not      very strong
3. H3K4me2, EZH2- the lncRNA parabola looks similiar but the mRNA has multiple humps/ dips up and down unlike the rest


#04

# load in chipseq data from analysis/01_peak_feautres

```{r loading in chipseq data}
load("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/01_create_consensus_peaks/results/peak_features.RData", verbose = T)
```

Next we want to load in our final_samplesheet from 01_DESEQ_counts.Rmd
# Reading in sample sheet

```{r read in sample_sheet}
# First let's read in the sample sheet to know what is what
samplesheet <- read_rds("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/05_RNAseq/01_differential_expression/results/final_samplesheet.rds")
```

# reading in TPM values from Salmon for our analyses

```{r reading in salmon Salmon TPMs}

# reading in salmon tpm
salmon_tpm <- read.csv("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/05_R_analyses/05_RNAseq/00_RNAseq_download_NF_core_pipeline/00_NF_CORE_RNAseq_Pipeline_run/run2/results/salmon/salmon_merged_gene_tpm.csv")

# TPM table is in same order as samplesheet
tpm <- salmon_tpm %>% 
  pivot_longer(cols = 2:ncol(.), names_to = "sample_id", values_to = "tpm") %>%
  merge(samplesheet) %>%
  group_by(gene_id, condition) %>%
  summarize(tpm = mean(tpm, na.rm = T)) %>%
  pivot_wider(names_from = condition, values_from = tpm, names_prefix = "tpm_")
```


# Abundance of genes in each cellular fraction

```{r TPM of genes in each fraction}
# First we need to the tpm DF into a matrix

tpm_matrix <- tpm %>% 
  column_to_rownames("gene_id") %>%
  as.matrix()
tpm_scaled <- t(scale(t(tpm_matrix)))
tpm_scaled <- tpm_scaled[complete.cases(tpm_scaled),]


# WARNING TAKES 5 min
# plotting
new.env()
pheatmap::pheatmap(tpm_scaled, show_rownames = FALSE)
pdf("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/figures/heatmap_expression.pdf", height =49, width = 12)
graphics.off()



peak_occurence_df <- read_csv("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/peak_occurence_dataframe.csv")


promoter_features_df <- merge(peak_occurence_df, tpm)

# saving this file
write.csv(promoter_features_df, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/results/promoter_feature_df_tpm.csv")
save(promoter_features_df, file = "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/results/promoter_feature_df.RData")

```
RESULT: 
(1) Most RNAs are abundant in the nucleus
(2) Some RNAs expressed in total that are not in other fractions




# Plotting binding versus expression

Now let's examine how binding effects expression.
We have published previously that the more binding
events at a promoter the more abundant the expression is.
Let's see if this holds for our current subset of DBPs
```{r DBP promoter binding versus total RNA expression}

# plotting binding vs total RNA expression
ggplot(promoter_features_df, 
            aes(y = log2(tpm_homo_sapiens_hepg2 + 0.001), x = number_of_dbp, color = gene_type)) + 
geom_point(data = promoter_features_df %>% filter(tpm_homo_sapiens_hepg2 > 0.001),
             shape = 17, alpha = 0.7) +
  geom_smooth(method = 'gam', formula = y ~ s(x, bs = "cs")) +
  #stat_cor() +
  geom_smooth(method = "lm") +
  
  
  # just making look pretty
  scale_x_continuous(expand = c(0,0)) +
  scale_color_manual(values = c("#a8404c", "#424242"), name = "Gene type") + 
  ggtitle("Expression vs. promoter binding events") + 
  xlab(expression('Number of TFs')) +
  ylab(expression(log[2](TPM))) 

ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/figures/binding_vs_expression_total_rna.pdf")
```
Result: 
(1) There is a linear trend with number of DBPS and expression levels
(2) There is a population of genes that have numerous DBPs with low expression

# What is the relationship between number of DBPS bound on a promoter versus RNA output (hint TPM)

The more DBPS bound the more expression. 

Looking at expression vs. promoter binding events there is a positive linear correlation for both lncRNA and mRNA. As the number of tfs increase the expression increases. Majority of the graph is bunched together by the classification of lncRNA or mRNA- so a lot of the black is together and a lot of the red is together. Out of both lncRNA and mRNA there is a smaller sub population that is not linear with majority of the trend- on the bottom with low expression even though they have high tfs. 

Looking at the nuclear expression vs. promoter binding events: there is also a positive linear trend as the number of tfs increases so does the expression and again we see a subpopulation that does not follow this trend. Looking at the cytomplasmic expression vs. promoter binding events there is also a linear trend and similiar results. But the linear line is at a lower expression compared to nuclear. Nuclear is around -3 log2 while cytoplasmic is -7 log2. The linear trend exists between nuclear and cytoplasmic but the expression is much lower in cytoplasmic. 

In the density plot for total rna- it can be seen that mRNAS have high expression compared to lncRNAs.
In the density plot for nuclear- mRNA also has a higher expression but the two are closer together compared to the total rna plot.


# Which subfraction of the cell has the highest expression.

The nuclear subfraction of the cell has the higher expression which can be seen from multiple different graphs such as the density plots and expression vs. binding event plots.



# Binding versus nuclear expression
Let's see if the binding versus expression holds in the nuclear fraction
```{r binding versus nuclear expression}

# Now let's make a similar plot for nuclear RNA abundance versus #DBPs bound to their promoter
ggplot(promoter_features_df, 
            aes(y = log2(tpm_homo_sapiens_nuclear_fraction + 0.001), x = number_of_dbp, color = gene_type)) + 
  geom_point(data = promoter_features_df %>% filter(tpm_homo_sapiens_nuclear_fraction > 0.001),
             shape = 17, alpha = 0.7) +
  geom_smooth(method = 'gam', formula = y ~ s(x, bs = "cs")) +
 # stat_cor() +
  
  
  #making look nice
  scale_x_continuous(expand = c(0,0)) +
  scale_color_manual(values = c("#a8404c", "#424242"), name = "Gene type") + 
  ggtitle("Nuclear Expression vs. promoter binding events") + 
  xlab(expression('Number of TFs')) +
  ylab(expression(log[2](TPM))) 

  # saving figure
  ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/figures/nuclear_expression-vs-promoter_binding.pdf")
```
RESULT: looks very similar to total RNA binding versus expression


# Binding versus cytoplasmic expression

Next we will determine the DBP binding versus cytoplasmic expression
```{Binding versus cytoplasmic expression}

# Same thing just seeing if there is a difference of cyto RNAs versus DBPs on promoter
ggplot(promoter_features_df, 
            aes(y = log2(tpm_homo_sapiens_cytosolic_fraction + 0.001), x = number_of_dbp, color = gene_type)) + 
  geom_point(data = promoter_features_df %>% filter(tpm_homo_sapiens_cytosolic_fraction > 0.001),
             shape = 17, alpha = 0.7) +
  geom_smooth(method = 'gam', formula = y ~ s(x, bs = "cs")) +
  #stat_cor() +
  
  
  # making look nice
  scale_x_continuous(expand = c(0,0)) +
  scale_color_manual(values = c("#a8404c", "#424242"), name = "Gene type") + 
  ggtitle("Cytoplasmic Expression vs. promoter binding events") + 
  xlab(expression('Number of TFs')) +
  ylab(expression(log[2](TPM))) 
  # saving figure
  ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/figures/cytoplasmic_expression-vs-promoter_binding.pdf")
  
```
RESULTS:
(1) everything seems to be low abundance 
(2) Some mRNAs are expressed in the nucleus -- we could look at this more later.
(3) The same linear trend holds but is driven by mostly low expression events.

# lncRNA versus mRNA expression in total RNA
Next we will directly test the lncRNA vs mRNA expression levels in total RNA. 

```{r determining lncRNA and mRNA expression levels in total RNA}

# plotting
ggplot(promoter_features_df, aes(x = log2(tpm_homo_sapiens_hepg2 + 0.01), color = gene_type))+
  geom_density()

# saving figure
ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/figures/mrna_lncrna_tpm_total_rna.pdf")

# let's also do the same for nuclear since lncRNAs are typically more nuclear
ggplot(promoter_features_df, aes(x = log2(tpm_homo_sapiens_nuclear_fraction + 0.01), color = gene_type))+
  geom_density()

# saving figure
ggsave("/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/figures/mrna_lncrna_tpm_nuclear.pdf")
```
RESULT:
(1) This yet again confirms lncRNAs have lower expression levels than mRNAs.
(2) In the nuclear fraction it shift's to closer.
(3) lot's of mRNA with nuclear expression -- that seems odd

# Let's make a heatmap of nuclear versus cytoplasmic expression

Looking at my heatmap of nuclear vs. cytoplasmic it shows that cystolic has less expression- there is way more 'blue' which correspond to lower values of expression. The blue is on top- corresponding to the right side of the cluster. Below that for the left side of the cluster the expression is much higher. On the half side of nuclear, the colors/expression is flipped on the cluster. The right side of the cluster is now mostly reds and yellows for a higher expression. When the left side of the cluster is lower expression of blue. 


# How many lncRNA and mRNA genes are sig in nuclear or cyto

When I look at the volcano plot- blue means significant while orange is not significant. It was found that cytoplasmic is more significant while the negative non significant is more nuclear. 

significant=(res_df$padj < .05)
significant
table(significant)

There are 14256 significant genes in nuclear and cytoplasmic. There are 15684 non significant genes.

# What is/are the most nuclear mRNA(s) -- is there a type of gene ontology associated with them?

protein coding have the highest nuclear fraction

# If we zoom in on high binding promoters (> 200 DBPs) are there any that don't have any expression?

high=filter(promoter_features_df,number_of_dbp > 200)
high$gene_name

Yes when the p value is zero- 

noRNA=(res_df$padj == 0)

table(noRNA)


We have previously observed that k562 cells also exhibit high binding promoters
that are not expressed. We termed them 'reservoirs' as they are a reservoir
for many Dna-protein interaction sites. Based on the plot above we observed that
this phenomena also exists in hepG2 cells as well. 

# Reservoirs : prelim code (we don't have enough DBPs yet -- next set of classes :)
Based on this we next wanted to identify the 'reservoirs' in hepG2 cells.
```{r defining HEPG2 reservoirs}

# first we will use a cutoff of 5 DBPs.
promoter_features_df$hepg2_reservoir <- 
  as.numeric(promoter_features_df$number_of_dbp > 5 & 
               promoter_features_df$tpm_homo_sapiens_hepg2 < 0.001)

# seeing what we got with table
table(promoter_features_df$hepg2_reservoir)

```
RESULT:
(1) There are no promoters with 5 elements bound


Now that we have defined reservoirs in hepG2 cells, we next want to determine how many 
are similar genomic regions in k562 and hepG2.

```{r reading in K562 reservoirs}

# reading in k562 promoter_features_DF
k562_df <- read_csv("/scratch/Shares/rinnclass/CLASS_2023/data/data/2020_k562_promoter_peak_df.csv")

# next we want to merge the k562 adn Hepg2 DFs 
k562_df <- k562_df %>% 
  dplyr::select(gene_id, reservoir, conservative_reservoir, tpm, expression, tf_binding, promoter_mean_tpm, promoter_median_tpm, promoter_max_tpm) %>%
  dplyr::rename(k562_reservoir = reservoir, 
                k562_conservative_reservoir = conservative_reservoir,
                k562_expression = expression,
                k562_tpm = tpm,
                k562_tf_binding = tf_binding,
                k562_promoter_mean_tpm =  promoter_mean_tpm,
                k562_promoter_median_tpm = promoter_median_tpm,
                k562_promoter_median_tpm = promoter_median_tpm,
                k562_promoter_max_tpm = promoter_max_tpm)

# save this file in new format
write_csv(k562_df,"/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/results/k562_df.csv")

# renaming promoter_features_df to hepg2_df
hepg2_df <- promoter_features_df %>%
  dplyr::select(gene_id, gene_name, tpm_homo_sapiens_hepg2, tpm_homo_sapiens_cytosolic_fraction, tpm_homo_sapiens_nuclear_fraction, tpm_homo_sapiens_insoluble_cytoplasmic_fraction, tpm_homo_sapiens_membrane_fraction, number_of_dbp, hepg2_reservoir) %>%
   
  dplyr::rename( tpm_total = tpm_homo_sapiens_hepg2,
                 tpm_cytosolic_fraction =  tpm_homo_sapiens_cytosolic_fraction,
                 tpm_nuclear_fraction = tpm_homo_sapiens_nuclear_fraction ,
                 tpm_insoluble_cytoplasmic_fraction = tpm_homo_sapiens_insoluble_cytoplasmic_fraction ,
                 tpm_membrane_fraction = tpm_homo_sapiens_membrane_fraction)

# let's save this handy file
write_csv(hepg2_df,"/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/results/hepg2_df.csv")
  
# Let's merge the k562 reservoirs in with HEPG2_df
# Merges on Gene_id
hepg2_k562_promoter_features_df <- merge(hepg2_df, k562_df)

# Now saving
write_csv(hepg2_k562_promoter_features_df, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/results/hepg2_k562_promoter_features_df.csv")

# Make a table of reservoir status
res_status <- hepg2_k562_promoter_features_df %>% 
  group_by(hepg2_reservoir, k562_reservoir, k562_conservative_reservoir) %>%
  summarize(count = n())

# saving for future
write_csv2(res_status, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/results/reservoir_overlap_stats.csv")
```
Result:
(1) There are 345 reservoirs in both K562 and HEPG2
(2) There are 80 Hepg2 reservoirs that overlap "conservative" k562 reservoirs



# Writting out files

```{r saving files for future use}
# We can now write these out for safekeeping / use in other analyses
write_csv(promoter_features_df, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/results/promoter_features_df.csv")
write_csv(tpm, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/results/mean_tpm_per_condition.csv")
write_csv(samplesheet, "/scratch/Shares/rinnclass/CLASS_2023/corrinbowman/CLASS_2023/CLASSES/06_final_going_global/analysis/04_binding_vs_expression/results/samplesheet.csv")
```




