---
title: "Final_class_project"
author: "Corrin Bowman"
date: "4/21/2023"
output: github_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

```
/scratch/Shares/rinnclass/CLASS_2023/data/data/peaks

# Your mission should you choose to accept it 
There is an enormous amount of data in ENCODE that 
needs to be analyzed. This includes 480 unique DBP
ChIPseq datasets. To facilitate your mission we provide you
1,066 peak files that have been run through the NF_CORE chipseq
pipeline. 

You will find all the .broadPeak peak files here:
/scratch/Shares/rinnclass/CLASS_2023/data/data/peaks

Use what you have learned in class to uncover
new results from this daunting data set. 
# Good luck the genome depends on you!


# first you might want to think about your organization
# 1) create a directory in your class folder for these analyses
    # -> folder name
          -> util/
            -> scripts.R
          -> analysis/
            -> 01_consensus_peaks
              -> .RMD
              -> results/
              -> figures
            -> 02_clustering etc....
  
  
# Consensus peaks
# Num_peaks_df
# Peak_occurence_df
# lncRNA and or mRNA promoters / gene annotations
# SAVE THIS !! save(object_name, nextobject, ...., file = "where you save this.Rdata")
# load(file.Rdata)


# First I want to analyze the large data for general properties 

# Filter out any chip data less 1,000 peaks == filtered consensus peaks

# How does peak number and genome coverage compare

looking at the plot of peak counts vs. total genome- there seems to be a correlation when the peak number is high the genome coverage is also high for example- ZNF280B, when the number of peaks is low the genome coverage is also lower. It looks pretty linear in the graph with some outliers. The genome coverage is a way bigger number, magnitudes bigger, than the peak numbers. This makes sense because the whole genome coverage isn't going to be covered in peaks.

# What is the distribution of promoter overlaps versus gene-bodies (hint hist)

hist(num_peaks_df$peaks_overlapping_promoters, breaks = 1000)
hist(num_peaks_df$peaks_overlapping_genebody, breaks = 1000)

When I look at the distribution of promoter overlaps it seems to be more evenly distributed across the x-axis. The majority of frequencies being 1, a lot are unique.

When I look at the distribution of gene body overlaps it is skewed to the left but much higher values than the promoter overlaps. The majority of frequencies are 1 meaning there is a lot of uniqueness.

# Make a list of genes that are "super binders" 

I am filtering the peak_occurence_df file in the number of dbp column above 200 to be considered a super binder which means over 200 dbps bind. 

super= filter(peak_occurence_df, number_of_dbp > 200) 

super$gene_name

There are so many names that are super binders which can be seen from the above code. 

# Is there a type of gene ontology associated with them versus the others?

I am going to index into the super binders list and see what kind of gene type they are and how many- 

table(super$gene_type)

What I found was there are two gene types associated with the super binders- lnRNA (2512) and protein coding (9184)

# Is there a difference in mRNA and lncRNA promoter overlaps?

When looking at the plot of number of DBPs and promoter overlaps for mRNA and lncRNA- mRNA definetely has more peaks overlapping promoters and has a greater slope than lncRNA. lncRNA may have less but seems more linear relationship than mRNA. There is 9,184 mRNA superbinders. Superbinders are more closely related to mRNA vs. lncRNA.


# Do lncRNAs also have super-binding promoters?

Yes they still have super binders there are 2512 superbinders but not as many as mRNA has.

# How many of these proteins are TFs? What is the most represented type of DBD?

table(num_peaks_df$tf)  

When I run the code indexing into the num_peaks file and then the tf column- there are 63 no and 344 yes

table(human_tfs$dbd)

C2H2 ZF is the most represented type of DBD and is seen 199 times- significantly more than the rest

```
# Results: I find that super-binders are more associated with mRNAs
# Results: I found X,000 mRNA superbinding promoters and Y lncRNA super-promoters


# Let's see how similar our genes are to each other
# Clustering 
```{r}

# What genes make sense to cluster together create an image (hint Pol II makes RNA)

I filtered through to get all the different gene types and found-  processed_pseudogene, lncRNA , unprocessed_pseudogene, miRNA, protein_coding.

It makes sense to cluster genes together that have similar functions/roles. For example all the protein coding genes together, and transcription factors together. 


# Find a cluster of genes your interested in and can figure out what those genes do -- are there unknown genes i there too? If so maybe one could hypothesize they have a similar function to the known genes (Hint ZNFs)

I looked at a smaller cluster with a few genes that are protein coding such as- smc3, suz12. Also in the cluster toward the right side is a few genes that are related to transcription factors such as- nfe2i2, maff and cebpb. There is a zinc finger gene(ZNF460) right next to the transcription factors and in this cluster so I guessed it was a function related to transcription factors but this gene is actually a protein coding one- so related to the other half of the cluster. 

Another example= ZNF883 is realted to cell carcinoma and is right next to gene EGR1 which is also associated with cancer- Leukemia.

I searched through a lot of the zinc finger genes and found one example that didn't seem to have much information on what it really does- ZNF614 is described as a protein coding gene realted to pathways for gene expression. The spot next door is gabpb1  which is protein coding related to gene expression. This one is involved in encoding the GA-binding protein transcription factor, beta subunit so maybe ZNF614 could be for something similiar with a beta subunit. 
 

# if we cluster by lncRNA and mRNA separately what are some similarities and differences?

Some similarities are they are both clustering so they have similiar shapes/figure. 
They both are pretty far apart/spread out- the clustering height is high
One difference is the mRNAs seem to be more closely related to eachother because they are closer vs. the lncRNAs seem slightly less related.

```
# Result: ZNFX seems to be similar to geneY and I hypothesize it might involved in A

# Metaplots
```{r}

# Let's look at the metaplot for all DBPs on lncRNA and mRNA promoters seperately (hint facet wrap).
Mega-Mega plot 

# Which genes seem to have a difference in where they bind on promoters between lncRNA and mRNA promoters

Looking at the big facet we have every gene in a plot with lncRNA as black and mRNA as red. Majority of all the graphs look similiar, a negative parabola with both lncRNA and mRNA pretty much overlapping eachother in the same spot/shape. There are a few exceptions that I noticed- 

1. H3K36me3- the shape was not a parabola but a bunch of squiggles, and lncRNA and mRNA don't overlap eachother really
2. H3K4me1, H3K79me2, H4K20me1- the parabola is flipped facing upwards rather down and lncRNA doesn't go down as far- the overlap is not      very strong
3. H3K4me2, EZH2- the lncRNA parabola looks similiar but the mRNA has multiple humps/ dips up and down unlike the rest

```


# RNAseq expression

```{r}

# What is the relationship between number of DBPS bound on a promoter versus RNA output (hint TPM)

The more DBPS bound the more expression. 

Looking at expression vs. promoter binding events there is a positive linear correlation for both lncRNA and mRNA. As the number of tfs increase the expression increases. Majority of the graph is bunched together by the classification of lncRNA or mRNA- so a lot of the black is together and a lot of the red is together. Out of both lncRNA and mRNA there is a smaller sub population that is not linear with majority of the trend- on the bottom with low expression even though they have high tfs. 

Looking at the nuclear expression vs. promoter binding events: there is also a positive linear trend as the number of tfs increases so does the expression and again we see a subpopulation that does not follow this trend. Looking at the cytomplasmic expression vs. promoter binding events there is also a linear trend and similiar results. But the linear line is at a lower expression compared to nuclear. Nuclear is around -3 log2 while cytoplasmic is -7 log2. The linear trend exists between nuclear and cytoplasmic but the expression is much lower in cytoplasmic. 

In the density plot for total rna- it can be seen that mRNAS have high expression compared to lncRNAs.
In the density plot for nuclear- mRNA also has a higher expression but the two are closer together compared to the total rna plot.

# Which subfraction of the cell has the highest expression.

The nuclear sufraction of the cell has the higher expression which can be seen from multiple different graphs such as the density plots and expression vs. binding event plots.

# Let's make a heatmap of nuclear versus cytoplasmic expression

Looking at my heatmap of nuclear vs. cytoplasmic it shows that cystolic has less expression- there is way more 'blue' which correspond to lower values of expression. The blue is on top- corresponding to the right side of the cluster. Below that for the left side of the cluster the expression is much higher. On the half side of nuclear, the colors/expression is flipped on the cluster. The right side of the cluster is now mostly reds and yellows for a higher expression. When the left side of the cluster is lower expression of blue. 


# How many lncRNA and mRNA genes are sig in nuclear or cyto

When I look at the volcano plot- blue means significant while orange is not significant. It was found that cytoplasmic is more significant while the negative non significant is more nuclear. 

significant=(res_df$padj < .05)
significant
table(significant)

There are 14256 significant genes in nuclear and cytoplasmic. There are 15684 non significant genes.

# What is/are the most nuclear mRNA(s) -- is there a type of gene ontology associated with them?

protein coding have the highest nuclear fraction

# If we zoom in on high binding promoters (> 200 DBPs) are there any that don't have any expression?

high=filter(promoter_features_df,number_of_dbp > 200)
high$gene_name

Yes when the p value is zero- 

noRNA=(res_df$padj == 0)

table(noRNA)

```
# Result in file.csv I have all the super binders that don't express RNA
